src/styles/sharedComponents/bottomSheet.js

```diff
@@ -8,10 +8,7 @@ const borderRadius = 24;
 
 const viewStyles: { [string]: ViewStyleProp } = StyleSheet.create( {
   bottomSheet: {
-    flex: 1,
-    paddingTop: 22,
-    paddingBottom: 70,
-    paddingHorizontal: 32
+    flex: 1
```

Comment:
should be able to use tailwind, i have not tested it on ios since the camera doesnt work on simulators.

but i can bring up the component on ios and test